### ISSUE TITLE
GDB stuck at "Detaching after fork from child process" when trying to load library

### ISSUE THREAD

**Issue by aaaasdfghjkllll** [lewisclark/glua-steal#63]
I'm using the x86_64 branch of Garry's Mod on Arch Linux. I have built libgluasteal.so from the latest source code but whenever I try injecting with gdb I get stuck at this:
```
(gdb) set $dlopen = (void*(*)(char*, int)) dlopen
(gdb) call $dlopen("/home/asdfghjkl/glua-steal/build/src/libgluasteal.so", 1)
[Detaching after fork from child process 53626]
```
I believe this is a gmod update because this has worked for me yesterday with the same build. I tried rebuilding and clean reinstalling gmod with no success.

**Comment by aaaasdfghjkllll** [comment]
32-bit branch works fine, so it is likely a x86_64 update which broke it

**Comment by aaaasdfghjkllll** [comment]
I feel like this could also be a problem with GModCEFCodecFix because I have tried downgrading and the game still has the same problem

**Comment by lewisclark** [comment]
> I feel like this could also be a problem with GModCEFCodecFix because I have tried downgrading and the game still has the same problem

Oh that's interesting. You went to a previously working version of the game and gluasteal still doesn't work?

**Comment by aaaasdfghjkllll** [comment]
Yes, I tried the one after the latest 32-bit release and it did not work
